### DISCUSSION TITLE
Android Boxes / Focus Issue

### DISCUSSION THREAD

**Discussion by khashayarsw** [react-native-tvos/react-native-tvos#535] [Q&A] [answered]
In some Android boxes that introduce themselves as mobiles, the focus of TouchableHighlight and TouchableOpacity components doesn't work. What solution do you suggest?

**Comment by akbari6776** [comment]
> In some Android boxes that introduce themselves as mobiles, the focus of TouchableHighlight and TouchableOpacity components doesn't work. What solution do you suggest?

I have the same problem.
When using the app on a mobile device or on Android boxes that mimic mobile behavior, we cannot handle the onFocus event for each component used as a button. This limitation prevents us from managing focus, making it challenging to determine the current focused item during navigation and apply any visual indication or styling. The only way to determine focus is by using onPress.

**Comment by douglowder** [comment, marked as answer]
Unfortunately, some boxes are not actually made with the Android TV OS, but are made with heavily customized versions of Android that do not support the Android TV APIs. This repo will not work on such systems.
